data/mods/DinoMod/monsters/zed-dinosaur.json

```diff
@@ -264,16 +269,17 @@
     "aggression": 100,
     "morale": 100,
     "speed": 100,
-    "melee_skill": 2,
+    "melee_skill": 3,
     "melee_dice": 1,
     "melee_dice_sides": 4,
     "melee_cut": 2,
     "armor_bash": 2,
     "armor_cut": 4,
     "armor_bullet": 2,
-    "hp": 70,
+    "hp": 528,
```

Comment:
From Google
Gallimimus/Mass
440 lbs
440 kg would be more than double that.  I was wondering since they are basically bear sized should they have 500 hp or do you think we are underestimating the hp a bear should have?  I'd be hard pressed to up a bear hp to this value since even today a number of people have hand to hand combatted a bear to death without guns.

Reply:
Brown bear or grizzly/Kodiac/polar?

Reply:
I've been going with wikipedia, I see huge variation in estimates too https://simple.wikipedia.org/wiki/Gallimimus